drivers/ethernet/eth_mcux.c

```diff
@@ -412,9 +412,11 @@ static void eth_mcux_phy_setup(void)
 	} else {
 		if (oms_override & PHY_OMS_NANDTREE_MASK) {
 			oms_override &= ~PHY_OMS_NANDTREE_MASK;
-			res = PHY_Write(ENET, phy_addr, PHY_OMS_OVERRIDE_REG, oms_override);
+			res = PHY_Write(ENET, phy_addr, PHY_OMS_OVERRIDE_REG,
+					oms_override);
 			if (res != kStatus_Success) {
-				LOG_WRN("Writing PHY register failed with status 0x%x", res);
```

Comment:
merge with previous patch and indent properly the lines.

Reply:
Indentation looks fine to me, can you explain me where it needs to be changed.

There are now 3 commits, do you want me to squash all of them into a single one or only squash the 2 last ones?

Reply:
The last 2.

And indentation at line 419 below is definitely wrong

Reply:
Can I put everything on a single line? That would means having that line longer that 80 characters.

Reply:
> Can I put everything on a single line? That would means having that line longer than 80 characters.

No, try to keep lines shorter than 80. If needed, just split them in suitable places.

Reply:
I shortened the message, it should now be fine.